pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopicPolicies.java

```diff
@@ -466,11 +466,18 @@ private class SetRetention extends CliCommand {
         private List<String> params;
 
         @Parameter(names = { "--time",
-                "-t" }, description = "Retention time in minutes (or minutes, hours,days,weeks eg: 100m, 3h, 2d, 5w). "
-                + "0 means no retention and -1 means infinite time retention", required = true)
+                "-t" }, description = "Retention time with optional time unit suffix "
+                + "eg: 100m, 3h, 2d, 5w. "
+                + "The suffix character can be s for seconds, m for minutes, h for hours, "
```

Comment:
> "The suffix character can be s for seconds, m for minutes, h for hours, d for days, w for weeks or y for years. "  

I think it is not necessary.

Reply:
Removed the extra line from description.

Reply:
Still in `CmdTopicPolicies`

Reply:
Updated based on recommendations.  Thanks